dev_tools/qualtran_dev_tools/all_call_graph.py

```diff
@@ -66,6 +66,8 @@ def keep(b: Bloq) -> bool:
         except DecomposeNotImplementedError:
             warnings.warn(f"{b} lacks a call graph.")
             return True
+        except ValueError:
+            return False
```

Comment:
what's this? Generally if `build_call_graph` causes a `ValueError` it's a real error, and the `all_call_graph` integration test should flag it

Reply:
Sorry. Actually, this change is unnecessary. I'm getting the list of Bloqs from the notebooks rather than from here.